hw-6/middlewares/check-auth-token.ts

```diff
@@ -0,0 +1,25 @@
+import { NextFunction, Request, Response } from 'express';
+import * as jwt from 'jsonwebtoken';
+import { HTTP_ERROR, JWT_SECRET } from '../constants/http-errors.enum';
+
+export const checkAuthToken = (req: Request, res: Response, next: NextFunction): void => {
+  const token = req.headers['x-access-token'] as string | undefined;
```

Comment:
We could put the header key under a constant to be able to use it here or somewhere else where needed.

Doing this, we improved significantly the maintainability of this feature.

Reply:
I extracted it to a constant. Thanks for raising this,